internal/scheduler/database/migrations/013_add_queue_scheduling_paused.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE queues ADD COLUMN scheduling_paused bool DEFAULT FALSE;
```

Comment:
I don't think this is doing anything. I'm almost certain this table isn't even used